.gitignore

```diff
@@ -1,4 +1,6 @@
+# misc
 .parrot
+package-lock.json
```

Comment:
```suggestion
```

Reply:
We wouldn't need this I think, with the `package-lock=false` in the `.npmrc`

Reply:
this would reduce the chance of package-lock being committed by mistake, better to be explicit